pkg/serviceaccount/tokens_controller.go

```diff
@@ -44,18 +44,16 @@ type TokensControllerOptions struct {
 	// SecretResync is the time.Duration at which to fully re-list secrets.
 	// If zero, re-list will be delayed as long as possible
 	SecretResync time.Duration
-}
 
-// DefaultTokenControllerOptions returns
-func DefaultTokenControllerOptions(tokenGenerator TokenGenerator) TokensControllerOptions {
```

Comment:
Default constructor functions are useful when defaulting fields but this isn't really doing anything. It's public but not godoced and it's name doesn't match what it actually produces (Token vs Tokens). I would just as soon leave it out until we want to default fields on TokensControllerOptions